src/KingSystem/Sound/sndMusicInfo.cpp

```diff
@@ -0,0 +1,54 @@
+#include "KingSystem/Sound/sndMusicInfo.h"
+
+namespace ksys::snd {
+
+MusicInfo::MusicInfo() : IParameterIO("mscinfo", 0) {}
```

Comment:
You might want to move this to the header. It's so small that it was probably defined inline.

i.e. `MusicInfo();` -> `MusicInfo() : IParameterIO("mscinfo", 0) {}`